domain/credential/state/types.go

```diff
@@ -193,3 +211,7 @@ type ownerAndCloudName struct {
 	OwnerName string `db:"owner_name"`
 	CloudName string `db:"cloud_name"`
 }
+
+type modelUUID struct {
+	UUID string `db:"uuid"`
```

Comment:
We've been using model.UUID here for the type

Reply:
Is this an established pattern? I have been using primitive types because that is what the driver works in. Conversion should be left to the state code and not that of the driver. At this level we want to work in the primitive type of the DDL not what the caller requires.